### ISSUE TITLE
Breaking a plant with produce will not yield produce.

### ISSUE THREAD

**Issue by smsunarto** [Terasology/SimpleFarming#9]
Breaking a plant with produce will only drop the seed without the produce.
Tested with TestBerry plant.

**Comment by Cervator** [comment]
Talked briefly on IRC but so we have a bit more info here: this is pretty much a gameplay / balancing issue. The testberry and blueberry are minimal implementations of plant growth, but not really adjusted for enjoyable and contiguous gameplay involving other systems :-)

Probably there should be two options:

* `e` click a plant to do its default thing - so `e` click a blueberry bush to pick a blueberry. Sensible. The idea of the "frob" button (default binding `e`) is to *start* with the target. So a chest opens an inventory, blueberry bush gives blueberries, etc
* *Alternative* click a plant optionally while holding a tool of some sort. Simplest example: right click a bush while holding a knife to harvest seeds instead of produce. Here the regular mouse click originates from the player and any equipped implement, not the bush. This action is done *to* the bush, not whatever it wants its default to be.

So in other words: if you want produce do one thing. If you want seeds do another thing. Breaking the plant should primarily just break the plant (pull it by its roots to get rid of it)

That's all IMHO as this is really more about gameplay desires than technical capabilities :-)

There is an ancient forum thread discussing sensible usage of mouse buttons and related bits, also covers the old radial menu that was never finished / merged: http://forum.terasology.org/threads/mouse-button-behavior.704

**Comment by jellysnake** [comment]
Closing this as it's not an issue anymore in the new system due to gameplay balancing
